LondonBritishLibrary/orient/BLorient611.xml

```diff
@@ -0,0 +1,249 @@
+<locus from="50r"/>
+<title type="complete" ref="LIT1295Dersan#HomilySane"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.9">
+<locus from="56r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyHamle"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.10">
+<locus from="60r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyNahase"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.11">
+<locus from="63r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyMaskaram"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.12">
+<locus from="66r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyTeqemt"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.13">
+<locus from="70r"/>
+<title type="complete" xml:lang="en">The miracles</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="85r"/>
+<title type="incomplete" xml:lang="en" ref="LIT2375Synaxa">Extracts from the Synaxarium regarding the festivals</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+</msItem>
+
+</msContents>
+
+<physDesc>
+<objectDesc form="Codex">
+  <supportDesc>
+      <support>
+          <material key="parchment"/>
+      </support>
+      <extent>
+          <measure unit="leaf">92</measure>
+          <measure unit="blank">Two unnumbered blank leaves at the end.</measure>
+          <dimensions type="outer" unit="in">
+              <height>11.5</height>
+              <width>9.625</width>
+              </dimensions>
+            </extent>
+    </supportDesc>
+
+    <layoutDesc>
+        <layout columns="2" writtenLines="19 23" cert="high">
+        </layout>
+    </layoutDesc>
+</objectDesc>
+
+<handDesc>
+
+    <handNote script="Ethiopic" xml:id="h1">
+    <locus from="5" to="84"/>
+    <date notBefore="1700" notAfter="1799" resp="PRS10171Wright"/>
+    </handNote>
+
+    <handNote script="Ethiopic" xml:id="h2">
+    <locus from="85" to="92"/>
+    <date notBefore="1800" notAfter="1899" resp="PRS10171Wright"/>
+    </handNote>
+
+</handDesc>
+
+<decoDesc>
+<summary></summary>
+<decoNote type="miniature" xml:id="d1">
+<locus target="#4v #84v"/>
+<desc>Pictures on <locus target="#4v">f. 4v</locus> and <locus target="#84v">84v</locus>.</desc>
+</decoNote>
+
+</decoDesc>
+
+<additions>
+<list>
+
+  <item xml:id="a1">
+  <locus target="#5r"></locus>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+            <placeName ref="INS0101MadhaneAlam"/>.</desc>
+  <q xml:lang="gez">ድርሳነ፡ ሚካኤል፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+  </item>
+
+<item xml:id="e1">
+<desc>Name of the first owner erased; later <persName role="owner" ref="PRS13517WalattaDawit"><foreign xml:lang="gez">ወለተ፡ ዳዊት፡</foreign></persName> and <persName role="owner" ref="PRS13518KeflaGiyorgis"><foreign xml:lang="gez">ክፍለ፡ ጊዮርጊስ፡</foreign></persName>,
```

Comment:
later owner? and I think the sentence is not complete

Reply:
OK, changed it slightly.